scrapper.py

```diff
@@ -35,7 +45,23 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
```

Comment:
is there a better way to handle those configuration items? especially given they are used several times